cico_setup.sh

```diff
@@ -59,8 +65,6 @@ push_image() {
         tag_push ${push_registry}/${image_repository}:latest ${image_name}
         tag_push ${push_registry}/${image_repository}:${short_commit} ${image_name}
     fi
-
-    echo 'CICO: Image pushed, ready to update deployed app'
```

Comment:
Unintentional change?

Reply:
I've shifted this echo part to the push image function so that we can verify if the push actually happened with 
if [ $? -eq 0 ] ...success else exit 1

Earlier, the success method used to be echoed regardless of whether a push happened successfully or not.